### PR TITLE
SO-417 - content changes supplier registration

### PR DESCRIPTION
**Pull request by adohall**

### CODE REVIEW

File: app/templates/components/seller-form/insurance-and-financial-document.hbs [hunk 1]
```diff
@@ -60,18 +60,17 @@
...
+  Buyers require this information in order to do business with you. By uploading it to your profile, buyers can access this information as they need it instead of requesting it from you. Don’t have these documents? Your insurer can issue a certificate of currency.
 </p>
-<br>
 <p>
-  <b> Upload files</b>
+  <b>Upload a file</b>
 </p>
 <p class="gray">
-  Files should be smaller than 10MB and in PDF, PNG or JPEG format. You can upload a single document or separate documents by using the 'Add another file' button below.
+  The file should be no larger than 5MB and in PDF, PNG or JPEG format. You can upload a single document or separate documents by using the 'Add another file' button below.
```
Comment (armansa) [inline app/templates/components/seller-form/insurance-and-financial-document.hbs]: 10MB

File: app/templates/components/seller-form/insurance-and-financial-document.hbs [hunk 2]
```diff
@@ -143,36 +141,35 @@
...
+  Don't have these documents? If you are unable to share the documents listed above, you can also ask your external accountant for a letter confirming your business can meet its debts when they're due and doesn't have any issues that could impact your ability to deliver your products or services.
 </p>
-<br>
 <p>
-  <b> Upload files</b>
+  <b> Upload a file</b>
 </p>
 <p class="gray">
-  Files should be smaller than 10MB and in PDF, PNG or JPEG format. You can upload a single document or separate documents by using the 'Add another file' button below.
+  The file should be no larger than 5MB and in PDF, PNG or JPEG format. You can upload a single document or separate documents by using the 'Add another file' button below.
```
Comment (armansa) [inline app/templates/components/seller-form/insurance-and-financial-document.hbs]: 10MB

File: app/templates/components/seller-form/accreditation-and-license.hbs [hunk 3]
```diff
@@ -12,35 +12,33 @@
...
   signal=form.signal
 }}
-<br>
 <p>
-  <b> Upload files</b>
+  <b> Upload a file</b>
 </p>
 <p class="gray">
-  Files should be smaller than 10MB and in PDF, PNG or JPEG format. You can upload a single document or separate documents by using the 'Add another file' button below.
+  The files should be no larger than 5MB and in PDF, PNG or JPEG format. You can upload a single document or separate documents by using the 'Add another file' button below.
```
[the remark this answers is not in the document]
Comment (adohall) [reply app/templates/components/seller-form/accreditation-and-license.hbs]: now changed

File: app/templates/components/seller-form/accreditation-and-license.hbs [hunk 4]
```diff
@@ -71,12 +69,11 @@
...
```
[the remark this answers is not in the document]
Comment (adohall) [reply app/templates/components/seller-form/accreditation-and-license.hbs]: now changed

**Pull request merged**